skfp/datasets/tdc/adme/adme.py

```diff
@@ -417,6 +499,33 @@ def load_cyp1a2_veith(
     verbose: bool = False,
 ) -> Union[pd.DataFrame, tuple[list[str]], np.ndarray]:
     """
+    Load the CYP1A2 subset of CYP P450 Veith dataset.
+
+    The CYP P450 genes are involved in the formation and breakdown (metabolism)
+    of various molecules and chemicals within cells.
+    The task is to predict CYP1A2 inhibition.
+    CYP1A2 localizes to the endoplasmic reticulum and its expression is induced by some polycyclic
+    aromatic hydrocarbons (PAHs), some of which are found in cigarette smoke.
+    It is able to metabolize some PAHs to carcinogenic intermediates.
+    Other xenobiotic substrates for this enzyme include caffeine, aflatoxin B1, and acetaminophen [1]_ [2]_.
+
+    This dataset is a part of "metabolism" subset of ADME tasks.
+
+    All CYP P450 Veith subsets:
```

Comment:
Does this render properly as a list without an empty line before first list item?